apps/account-api/k6-test/account-service-load.k6.skip.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
I removed all the airbnb eslint packages, and so none of these lint-disables were being triggered. We can consider putting some of these back in the actual eslint config, but some of them, we seemed to always want to ignore and so there was no point to the lint rule. Others lived only in specs, so maybe those are fine not being a rule for us.